netebpfext/net_ebpf_ext.c

```diff
@@ -144,9 +179,32 @@ _net_ebpf_ext_resource_release_classify(
     uint64_t flow_context,
     _Inout_ FWPS_CLASSIFY_OUT* classify_output);
 
+static void
+_net_ebpf_ext_flow_established_classify(
```

Comment:
Missing function definition

Reply:
Thanks, I added the function definition in my local repo, will push these changes after implementing the other feedback. I will add the logic into the functions once I start implementing the flow established and mac hooks.

Reply:
Are you planning to add those implementation with a seperate PR?

Reply:
I will add it to this PR, I just wanted to get your feedback (like the comment and spacing one) and so I can implement that too before I push again.